src/chains/ethereum/ethereum/tests/forking/block.test.ts

```diff
@@ -68,5 +68,12 @@ describe("forking", function () {
       const block = await provider.send("eth_getBlockByNumber", ["0x0", true]);
       assert.deepStrictEqual(block, block0);
     });
+
+    it("should get TransactionCountByHash from the original chain", async () => {
```

Comment:
```suggestion
    it("should get transaction count by hash from the original chain", async () => {
```
Just to be more in line with how other tests are formatted!